Rakefile

```diff
@@ -1,3 +1,5 @@
+require "pact/tasks"
```

Comment:
This fix worked for https://github.com/alphagov/account-api/pull/44, https://github.com/alphagov/frontend/pull/2715, https://github.com/alphagov/collections/pull/2358.

But not for https://github.com/alphagov/publishing-api/pull/1937 - which is Pact-tested against Content Store - suggesting some sort of link.